### ISSUE TITLE
[Chapters of the book] Improvements in CTA

### ISSUE THREAD

**Issue by legendarykamal** [service-mesh-patterns/service-mesh-patterns#83]
#### Description

1> The icon is very small, make it bigger 
![image](https://user-images.githubusercontent.com/95926324/193725034-00af10f9-8e13-43ea-a058-167963b73c85.png)

2> There are 2 duplicate CTA's we need to keep only 1.
#### Environment:
- Meshery Version: 
- Kubernetes Version: 
- Host OS:
- Browser:

**Comment by legendarykamal** [comment]
@leecalcote @Nikhil-Ladha should be make the second CTA redirect to slack?  as both are redirecting to same place.

**Comment by sam1code** [comment]
I would love to work on this @legendarykamal .

**Comment by leecalcote** [comment]
@sam1code it'd be good to see the other four that you have assigned completed first (or at least of all them under PR review).

**Comment by sam1code** [comment]
Yes only one remaining I'll be creating pr of the kat one in some times

**Comment by AbhishekTiwari23** [comment]
Can I work over this if it still exist?

**Comment by Nikhil-Ladha** [comment]
You can work on this issue @AbhishekTiwari23 .
Though, not that the requirement is to change to callout to "Layer5 Dlack Workspace" with the slack icon (whitewashed) in the beginning (just like now).

**Comment by AbhishekTiwari23** [comment]
@legendarykamal sir, can you please share the ink for that page where I have to make changes.
It will be thankful for me.

**Comment by AbhishekTiwari23** [comment]
Is it this one ?
<img width="1440" alt="Screenshot 2022-10-14 at 12 45 15 AM" src="https://user-images.githubusercontent.com/97469132/195689343-4f3748f6-f992-4199-8501-fbdd2bcbfabe.png">

**Comment by AbhishekTiwari23** [comment]
Oh, now I get it. I apologise for the inconvenience

**Comment by AbhishekTiwari23** [comment]
Hello @legendarykamal @Nikhil-Ladha, sir, this is my first time when am contributing to this repo, and I am trying but not able to set it up on localhost. Can u please guide me? I followed all the steps of contributing.MD file, and there are no steps mentioned for this. I will be thankful if you guide me.

**Comment by Nikhil-Ladha** [comment]
Just navigate to the `site` directory and execute the `npm install` and `npm start` commands. That should do the job.

**Comment by AbhishekTiwari23** [comment]
https://user-images.githubusercontent.com/97469132/196407085-ad1a9945-4ba7-45a6-b3cc-35f61babc3f7.mov

Is it fine?

**Comment by Nikhil-Ladha** [comment]
@debo19 I forgot your suggestion for the callout on this page, can you please re-iterate here?

**Comment by debo19** [comment]
Just change the content on the second callout to point to something else like [layer5.io/projects](https://layer5.io/projects) or [layer5.io/learn/learning-paths](https://layer5.io/learn/learning-paths)

**Comment by leecalcote** [comment]
@UtkarshMishra12

**Comment by eeshaanSA** [comment]
@AbhishekTiwari23 are you still working on this?

**Comment by eeshaanSA** [comment]
Is this issue still valid? @legendarykamal

**Comment by eeshaanSA** [comment]
@UtkarshMishra12 not able to reproduce. Closing. Open again if you feel this issue needs more attention.
